Makefile

```diff
@@ -254,6 +254,11 @@ start-stress-test: zetanode
 	@echo "--> Starting stress test"
 	cd contrib/localnet/ && $(DOCKER) compose --profile stress -f docker-compose.yml up -d
 
+start-migrate-test: zetanode
```

Comment:
Can we call this `tss-migration-test` or something more specific?

Please add it to the matrix in `.github/workflows/e2e.yml`

Reply:
1ffe90727cffc72f20c38c8c510bc2d9d490b287